go.mod

```diff
@@ -36,7 +36,7 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
 
 	// Pin RHsyseng library versions
-	github.com/RHsyseng/console-cr-form => github.com/RHsyseng/console-cr-form v0.0.0-20200414161125-135bc9b52976
+	github.com/RHsyseng/console-cr-form => github.com/RHsyseng/console-cr-form v0.0.0-20210323180350-be2aa15abde0
```

Comment:
can you please remove the console update from this PR and use the other one you've created last week?

Reply:
Ok

Reply:
ok